runner.py

```diff
@@ -143,13 +149,17 @@ def main(
     finished_pipes = start_pipe_threads(pipe_commands, logger)
 
     # Move selected files to report dir
-    logger.info(f'Moving files to {config.get("general", "report_dir")}')
-    transferred_files = report_results(finished_pipes, outdir, sample_name, config)
-    for pipe, transfers in transferred_files.items():
-        logger.info(f"Moved {transfers} files generated by {pipe}")
+    if skip_report:
+        logger.info('Reporting disabled, skipping copy to report dir')
+    else:
+        report_dir = os.path.join(config.get("general", "report_dir"), sample_name)
+        logger.info(f'Moving files to {report_dir}')
```

Comment:
These single quotes should also be double quotes

Reply:
Fixed in ee29a39